src/content/development/contribution-guide/_index.en.md

```diff
@@ -10,23 +10,30 @@ in order for their contributions to the project to be addressed in a timely mann
 
 ## Project Resources
 
-To be expanded
+Submariner uses [GitHub Projects](https://docs.github.com/en/issues/organizing-your-work-with-project-boards) to manage releases. Read
+“[Tracking progress on your project board](https://docs.github.com/en/issues/organizing-your-work-with-project-boards/tracking-work-with-project-boards)"
+to learn more on how to work with projects.
 
 ### Backlog Board
 
-To be expanded
+[The backlog board](https://github.com/orgs/submariner-io/projects/15) hosts issues or features that are not scheduled
+for the release currently being developed.
 
 ### Current Release Board
 
-To be expanded
+Current release work is tracked in [the 0.13 board](https://github.com/orgs/submariner-io/projects/20).
```

Comment:
Do we want to link directly to the 0.13 board? We will have to remember to update it each version :)

Reply:
What else can we do?

Reply:
Just link to the projects page?